documentation/synthetic_tokens/tutorials/liquidation.md

```diff
@@ -0,0 +1,420 @@
+If you have a wallet mnemonic already you can skip this section.
+
+To generate a new mnemonic you can run the following from the `/core` directory:
+
+```bash
+node -e "console.log(require('bip39').generateMnemonic())"
+# your mnemonic should print here
+```
+
+You can then load this mnemonic into truffle and view the associated address.
+To do this set the mnemonic as an environment variable by running:
+
+```bash
+# Add the new mnemonic to your environment variables. Be sure to replace with your mnemonic.
+export MNEMONIC="sail chuckle school attitude symptom tenant fragile patch ring immense main rapid"
+
+# Start the truffle console
+$(npm bin)/truffle console --network kovan_mnemonic
+
+# Print the address of your newly created account
+accounts[0]
+0x45Bc98b00adB0dFe16c85c391B1854B706b7d612
+```
+
+You can now fund this wallet with the associated currency for the type of bot you want to run.
+To learn more about creating synthetic tokens to back fund your liquidation bot see [this](./using_the_uma_sponsor_cli_tool.md) tutorial.
+
+### Creating a price feed API key
+
+All bots require a price feed to inform their liquidation decisions.
+The easiest price feed to intergrade into is [CryptoWatch](https://cryptowat.ch/). To create an API Key do the following:
+
+1. Create an account [here](https://cryptowat.ch/account/create).
+2. Generate an API key [here](Generate an API key).
+
+Keep this key handy. You'll need it when configuring the bots.
+
+## Running the liquidator and disputer bots locally
+
+This tutorial start with running the liquidator and disputer bots locally from your machine without docker or any complex execution environment. This is meant to be the simplest way posable to start up a bot.
+
+**a) Configuring environment**
+To start a bot the first step is to configure the bot's settings.
+Liquidation bots require 4 main configurations settings which are configured using environment variables.
+To set this up create a `.env` file in the `/core` directory of the repo it:
+
+```bash
+POLLING_DELAY=30000
+EMP_ADDRESS=0xDe15ae6E8CAA2fDa906b1621cF0F7296Aa79d9f1
+MNEMONIC=sail chuckle school attitude symptom tenant fragile patch ring immense
+PRICE_FEED_CONFIG={"type":"medianizer","apiKey":"YOUR_API_KEY","pair":"ethbtc","lookback":7200,"minTimeBetweenUpdates":60,"medianizedFeeds":[{"type":"cryptowatch","exchange":"coinbase-pro"},{"type":"cryptowatch","exchange":"binance"},{"type":"cryptowatch","exchange":"bitstamp"}]}
+```
+
+The parameters above, as well as other optional parameters are explained in the appendix of this tutorial. **Be sure to add in your mnemonic and your crypto watch API key.** The parameter in the example above conform to [UMIP-2](<[../..](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-2.md#implementation)>)'s specification.
+
+**b) Starting the bots**
+
+Now that your env is set up you can run the bot. Run the following command from the `core` directory to start the bots on Kovan:
```

Comment:
You should make a note that the user will need Kovan ETH or the bot will crash when trying to send the first transaction (an approval to spend collateral currency). User can go here: https://faucet.kovan.network/

Also, should make a note ideally where to acquire Kovan DAI

Reply:
@chrismare

Reply:
I committed a change to this